src/FileCacheManager.php

```diff
@@ -32,144 +35,91 @@
  */
```

Comment:
Sorry. What about describing caching logic in general docblock for FileCacheManager ?
As it's now a little more complicated